app/javascript/components/widgets/twitter/panel.jsx

```diff
@@ -61,54 +61,16 @@ export const TabBar = () => (
   </svg>
 );
 
-const Twitter = ({ tweets }) => {
-  const previous = takeLast(1, splitAt(1, tweets))[0];
-  return (
-    <PanelWrapper>
-      <Tweet tweet={tweets[0]} isPrimary />
-      <PreviousWrapper>
-        <TweetDivider>Previous Tweets</TweetDivider>
-        <TabBar />
-        {previous.map(tweet => (
-          <React.Fragment key={tweet.createdAt}>
-            <Tweet tweet={tweet} isPrimary={false} />
-            <TabBar />
-          </React.Fragment>
-        ))}
-      </PreviousWrapper>
-    </PanelWrapper>
-  );
-};
-
```

Comment:
Nice, I think combining this Twitter component with its parent component below is much clearer because the parent component isn't named (the parent component was the anonymous function `export default () => ( ...`)